src/js/actions/layereffects.js

```diff
@@ -325,6 +333,89 @@ define(function (require, exports) {
     setShadowSpread.reads = [locks.PS_DOC, locks.JS_DOC];
     setShadowSpread.writes = [locks.PS_DOC, locks.JS_DOC];
 
+    /**
+     * Copies the layer effects of the source layer to all the layers
```

Comment:
The problem with "copy" is that it implies that it is not a destructive action. Copying something in a document does not change the document, but this does. Also, we'll soon have an actual "copy effects" command, which will do an actual non-destructive copy. How about "duplicate"? This would be analogous to Photoshop's layer duplicate command.